web/ui_utils.js

```diff
@@ -381,6 +381,15 @@ function noContextMenuHandler(evt) {
   evt.preventDefault();
 }
 
+function isFileSchema(url) {
+  if (!url.__proto__.startsWith) {
```

Comment:
Since this is polyfill code, it shouldn't be here but in `src/shared/compatibility.js` instead. It should be in the same format as https://github.com/mozilla/pdf.js/blob/master/src/shared/compatibility.js#L189 with the `core-js/fn/string/starts-with` function (see https://github.com/zloirock/core-js/blob/master/fn/string/starts-with.js).

Reply:
@timvandermeij , sure I'll make the changes

Reply:
@timvandermeij , please have a look now.